client/src/App.tsx

```diff
@@ -1,23 +1,21 @@
-import { observer } from "mobx-react-lite"
-import { authStore } from "./stores/AuthStore"
+import { observer } from "mobx-react-lite";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
```

Comment:
added react-router-dom without adding it to the package.json (use --save when installing)